contracts/RiskSteward/RiskStewardReceiver.sol

```diff
@@ -0,0 +1,322 @@
+     */
+    IRiskOracle public immutable RISK_ORACLE;
+
+    /**
+     * @notice Address of the CorePool comptroller used for selecting the correct comptroller abi
+     */
+    ICorePoolComptroller public immutable CORE_POOL_COMPTROLLER;
+
+    /**
+     * @notice Mapping of market and update type to last update. Used for debouncing updates.
+     */
+    mapping(bytes marketUpdateType => uint256) public lastProcessedTime;
+
+    /**
+     * @notice Mapping of processed updates. Used to prevent re-execution
+     */
+    mapping(uint256 updateId => bool) public processedUpdates;
+
+    /**
+     * @notice Time before a submitted update is considered stale
+     */
+    uint256 public constant UPDATE_EXPIRATION_TIME = 1 days;
+
+    /**
+     * @notice Event emitted when an update is successfully applied
+     */
+    event RiskParameterUpdated(uint256 updateId);
+
+    /**
+     * @notice Flag for if a risk parameter type can be updated by the steward
+     */
+    error ConfigNotActive();
+
+    /**
+     * @notice Error for when an update was not applied within the required timeframe
+     */
+    error UpdateIsExpired();
+
+    /**
+     * @notice Thrown when an update has already been processed
+     */
+    error ConfigAlreadyProcessed();
+
+    /**
+     * @notice Thrown when the debounce period hasn't passed for applying an update to a specific market/ update type
+     */
+    error UpdateTooFrequent();
+
+    /**
+     * @notice Thrown when the new value of an update is our of range
+     */
+    error UpdateNotInRange();
+
+    /**
+     * @notice Thrown when a config that is not implemented is set
+     */
+    error UnsupportedUpdateType();
+
+    /**
+     * @notice Flag to pause/unpause update processing
+     */
+    bool public paused;
+
+    /**
+     * @notice Event emitted when pause state is changed
+     */
+    event PauseStateChanged(bool newState);
+
+    /**
+     * @notice Error thrown when contract is paused
+     */
+    error ContractPaused();
+
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address riskOracle_, address corePoolComptroller_) {
+        require(riskOracle_ != address(0), "Risk Oracle address must not be zero");
+        require(corePoolComptroller_ != address(0), "Core Pool Comptroller address must not be zero");
+        RISK_ORACLE = IRiskOracle(riskOracle_);
+        CORE_POOL_COMPTROLLER = ICorePoolComptroller(corePoolComptroller_);
+        _disableInitializers();
+    }
+
+    function initialize() external initializer {
+        __Ownable2Step_init();
+    }
+
+    /**
+     * @notice Sets the risk parameter config for a given update type
+     * @param updateType The type of update to set the config for
+     * @param debounce The debounce period for the update
+     * @param maxIncreaseBps The max increase bps for the update
+     * @param isRelative Whether the max increase bps is relative or absolute
+     */
+    function setRiskParameterConfig(
```

Comment:
We could extend `AccessControlledV8`, and integrated the ACM here. This way, the timelock contracts could invoke `setRiskParameterConfig`